topo/fot/forch/forch.yaml

```diff
@@ -20,6 +20,11 @@ orchestration:
     port_description: TAP
     sequester_timeout_sec: 360
     default_auto_sequestering: disabled
+    test_result_device_state:
+      - result: FAILED
+        device_state: infracted
+      - result: PASSED
```

Comment:
I'd say either { test_result, device_state } or { result, state } for consistency.